docker-ci.sh

```diff
@@ -48,8 +48,8 @@ function run_test {
     [ -n "$PERF" ] && url="$url&include=DevExpress.performance&workerInWindow=true"
 
     if [ -n "$TZ" ]; then
-        echo "Time-Zone: $TZ"
```

Comment:
How to make sure that that the desired timezone is applied correctly?

Reply:
`dpkg-reconfigure --frontend noninteractive tzdata` show UTC and applied date

Reply:
Got it, tnx!
![image](https://user-images.githubusercontent.com/1420883/73938097-9f013580-48f7-11ea-8b13-306b9a6991a0.png)